src/textual/message_pump.py

```diff
@@ -655,6 +658,16 @@ async def _dispatch_message(self, message: Message) -> None:
             # Allow apps to treat events and messages separately
             if isinstance(message, Event):
                 await self.on_event(message)
+            elif "debug" in self.app.features:
+                start = perf_counter()
+                await self._on_message(message)
+                if perf_counter() - start > SLOW_THRESHOLD_MS / 1000:
+                    log.warning.verbosity(message.verbose)(
+                        f"method=<{self.__class__.__name__}.{message.handler_name}>",
+                        ">>>",
+                        f"Took over {SLOW_THRESHOLD_MS}ms to process.",
+                        "\nTo avoid screen freezes use the [work decorator](/guide/workers/#workers)"
```

Comment:
This goes to the textual console, which doesn't support Markdown syntax. Suggest simply "Consider using a worker."